getting-started/README.md

```diff
@@ -205,6 +237,149 @@ console.log(`Contract address: ${greeter.address}`)
 await greeter.greet()
 ```
 
+<!--
```

Comment:
Can we nuke this? Why comment it out?

Reply:
Because I was working on it. But I'll nuke it and store it in the Linear ticket.